kin-sdk/kin-sdk-lib/src/test/kotlin/kin/sdk/internal/queue/PaymentQueueManagerTest.kt

```diff
@@ -0,0 +1,170 @@
+package kin.sdk.internal.queue
+
+import com.nhaarman.mockitokotlin2.*
+import kin.base.KeyPair
+import kin.sdk.internal.data.PendingBalanceUpdater
+import kin.sdk.internal.events.EventsManager
+import kin.sdk.queue.PendingPayment
+import org.hamcrest.MatcherAssert.assertThat
+import org.hamcrest.Matchers.equalTo
+import org.junit.Before
+import org.junit.Test
+import java.math.BigDecimal
+
+class PaymentQueueManagerTest {
+
+    object Constants {
+        const val ONE_MILLI = 1000L
+        const val DELAY_BETWEEN_PAYMENTS_MILLIS = 3000L
```

Comment:
please reduce the values in order to shorten the tests time (faster unit tests)

Reply:
to how much?

Reply:
you can go for the 100s of millis for the "delay between payments", that safe enough I think.